dev.sh

```diff
@@ -42,4 +83,5 @@ rm -rf workspace
 
 sleep 2s
 docker logs reverse-proxy
-echo "Run 'docker stop reverse-proxy montagu-metrics' to stop"
\ No newline at end of file
+echo "Run 'docker stop reverse-proxy montagu-metrics' to stop proxy containers"
```

Comment:
A nicer way of handling this might to end with `sleep infinity` and let a `ctrl-c` bring down the containers, as in this script:
https://github.com/vimc/montagu-webapps/blob/master/app/scripts/run-development-apis.sh#L8

We can also automatically bring the network/containers down on error with something like:
https://github.com/vimc/montagu-webapps/blob/master/app/scripts/run-integration-tests-with-apis.sh#L17
which is handy if it fails because e.g. ports are already allocated